ci/container/test_component.sh

```diff
@@ -66,9 +66,17 @@ env | grep SNOWFLAKE_ | grep -v PASS
 
 echo "[INFO] Starting hang_webserver.py 12345"
 python3 $THIS_DIR/hang_webserver.py 12345 > hang_webserver.out 2>&1 &
-MOCHA_CMD=(
-    "mocha" "--timeout" "$TIMEOUT" "--recursive" "--full-trace"
-)
+
+if [[ "$SHOULD_GENERATE_COVERAGE_REPORT" -eq "1" && "$CLOUD_PROVIDER" == "AWS" ]];
+  then
+    MOCHA_CMD=(
+       "npx" "nyc" "mocha" "--exit" "--timeout" "$TIMEOUT" "--recursive" "--full-trace"
```

Comment:
`npx nyc` is only a prefix? so maybe we can just in the if logic add it to the `MOCHA_CMD` and do not copy all parameters (`--recursive`, etc)?